components/03-modules/header/header.hbs

```diff
@@ -1,17 +1,95 @@
-<header class="header container">
-    {{ render '@logo' }}
+<header class="header">
+    {{ render '@marketing-bar' }}
+    <div class="header__top-bar">
+        <div class="container header__top-bar-wrapper">
+            <div class="header__contact-info">
+                + 48 68 433 00 00
+                <a class="header__contact-info-link" href="#">Contact</a>
+            </div>
+            <div class="header__links">
+                <a class="header__links-item" href="#">
```

Comment:
Try extracting data to config.js and using loop for this `header__link-item`

Reply:
done.